docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -1,34 +1,57 @@
-==========================
-File system access control
-==========================
+=========================
+File-based access control
+=========================
 
-This access control plugin allows you to specify authorization rules in a JSON file.
+To secure access to data in your cluster, you can implement file-based access
+control where access to data and operations is defined by rules declared in
+manually-configured JSON files.
+
+There are two types of file-based access control:
```

Comment:
There are two types of access control in general (not specific to file based). Maybe we should mention this somewhere (not in this file though since it's specific to the file-based mechanisms).

Reply:
Currently that's described in /security/overview, which seems like the right place for it. I don't think any immediate edits need to be made there, but am open to suggestions.

Reply:
/security/overview satisfies my ask. Resolved.